MsTestToXunitConverter/AssertRewriter.cs

```diff
@@ -11,7 +12,7 @@ namespace MsTestToXunitConverter
     /// </summary>
     internal static class AssertRewriter
     {
-        private static InvocationExpressionSyntax InvocationRewriter(this InvocationExpressionSyntax invocation, string from, string to, IdentifierNameSyntax identifier = null)
+        private static InvocationExpressionSyntax InvocationRewriter(this InvocationExpressionSyntax invocation, string from, string to, IdentifierNameSyntax identifier = null, Func<InvocationExpressionSyntax, InvocationExpressionSyntax> func = null)
         {
             if (invocation.Expression.Kind() != SyntaxKind.SimpleMemberAccessExpression)
             {
```

Comment:
re line 22: I don't understand why "from" is an argument to this method.

Reply:
So you can go `from` something `to` another thing. e.g.: from `Assert.Fail` to `True`